new/LIT7292MartyrMa.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
```

Comment:
are you sure that this file belongs in this branch? I think you have confused something.